pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "codemodder-python"
+version = "0.57.0"
+requires-python = ">=3.9.0"
+readme = "README.md"
+license = {file = "LICENSE"}
+dependencies = [
```

Comment:
I assume these are the dependencies that are currently in requirements/codemodder.txt? Or what is the relationship between this and that file? Can we delete the file?

Reply:
Yes we can delete it 👍